### PR TITLE
Pushbrace height [AB#1553104]

### PR DESCRIPTION
**Pull request by phillipdlai**

### CODE REVIEW

File: resources/schema/spidacalc/calc/structure.schema [hunk 1]
```diff
@@ -794,6 +794,9 @@
           "attachmentHeight": {
             "$ref": "../../general/measurable.schema"
           },
+          "height": {
```
Comment (jasongarrett) [inline resources/schema/spidacalc/calc/structure.schema:797]: either height should be required by the schema or we should be able to handle json that does not include the push brace height

Comment (phillipdlai) [reply resources/schema/spidacalc/calc/structure.schema:798]: does this mean that we don't want to run PushBraceHeightChangeSet.applyToStructure?

Comment (phillipdlai) [reply resources/schema/spidacalc/calc/structure.schema:797]: we won't need PushBraceHeightChangeSet.applyToResults and applyToStructure anymore right?

Comment (jasongarrett) [reply resources/schema/spidacalc/calc/structure.schema:797]: Right.

**Review by jasongarrett — APPROVED** [review]

**Pull request merged**